.eslintignore

```diff
@@ -3,4 +3,7 @@ foundry.iml
 
 node_modules
 
-dist
\ No newline at end of file
+dist
+
+# Jest
+jest.config.js
```

Comment:
is this necessary? config files should still be linted?

Reply:
The config files normally follow the tool format convention, so I don't see advantage to have lint warnings on them.

Reply:
is it because it's CJS and eslint checks for ESM?

Reply:
Yes, for example if I activate the scan for the `jest.config.js` I got that kind of error:

```
C:\Dev\foundry\foundry\packages\create-workleap\jest.config.js
  1:1  error  'module' is not defined  no-undef
```

Reply:
Est-ce qu'on pourrait mettre ce fichier là en ts à la place? 

![image](https://user-images.githubusercontent.com/38871812/231798097-dc5fe604-a79c-48e7-a567-71ee1310fd55.png)
https://jestjs.io/docs/27.x/configuration

Reply:
Seems to work nicely 👍